far2l/plist.cpp

```diff
@@ -48,6 +48,9 @@ static BOOL KillProcess(DWORD dwPID);
 
 void ShowProcessList()
 {
+    // Currently unsupported on linux
+    return;
```

Comment:
why return here? its not called anyway

Reply:
called by Ctrl+W

Reply:
> why return here? its not called anyway

still callable by Ctrl+W

Reply:
may be just run htop?

Reply:
or user script (like roots.sh) that will run htop if it installed, otherwise - top..

Reply:
> or user script (like roots.sh) that will run htop if it installed, otherwise - top..

the problem is Ctrl+W can be pressed in viewer/editor also

what should we do in those cases?